### ISSUE TITLE
set input delay...

### ISSUE THREAD

**Issue by Apprisco** [spyoungtech/ahk#335]
### Checked the documentation

- [X] I have checked [the documentation](https://ahk.readthedocs.io/en/latest/api/methods.html) for the feature I am requesting

### describe your feature request

Hi,
it is completely unclear whether or not you can setup input delay.
Right now it types a text message in mere milliseconds, but i want to put a key delay after each stroke for 10ms.
This isn't supported right now it seems, while it's a basic feature in AHK.

**Comment by spyoungtech** [comment]
Have you tried using the `key_delay` parameter?

**Comment by Apprisco** [comment]
where? Key_delay exists for just typing, but not sending messages TO a window.

**Comment by spyoungtech** [comment]
That's true. There is an oversight in the API for `Window.send` and this parameter isn't there. Though, you did not mention your use case is for sending to a window until just now :)

I should be able to add this feature quickly, both for a `key_delay` parameter in `Window.send` and also a global `set_key_delay` perhaps.

**Comment by Apprisco** [comment]
This is great! By the way, is this send basically ControlSend? Would it cause the same issues as win,shift,alt,ctrl keys being stuck?

**Comment by spyoungtech** [comment]
Right. The underlying functionality of `Window.send` uses AutoHotkey's `ControlSend`, so the behavior you observe will be identical to that of `ControlSend` in AutoHotkey scripts.
